src/main/drivers/pwm_output.c

```diff
@@ -280,3 +281,28 @@ void servoInit(const servoConfig_t *servoConfig)
 }
 
 #endif
+
+#ifdef BRUSHED_ESC_AUTODETECT
+uint8_t hardwareMotorType = MOTOR_UNKNOWN;
```

Comment:
@MJ666 : Quick grep of code shows that almost all enums are `typedef enum` and `_e` suffix is more common ...